linux_os/guide/system/logging/rsyslog_sending_messages/rsyslog_remote_loghost/rule.yml

```diff
@@ -16,13 +16,13 @@ description: |-
     they may not be supported in all environments.
     <br />
     To use UDP for log message delivery:
-    <pre>*.* @<i>loghost.example.com</i></pre>
+    <pre>*.* @<i><sub idref="rsyslog_remote_loghost_address" /></i></pre>
```

Comment:
Can you do the same change on line 13, please?

Reply:
I'll fix shortly.